Formula/packr.rb

```diff
@@ -56,6 +56,11 @@ def install
 
     system "go", "mod", "init", "example"
     system bin/"packr"
+    # Bypass `go get` check. Needed for go 1.16
+    touch "goget_packr.sh"
+    chmod 0755, "goget_packr.sh"
+    inreplace "goget_packr.sh", "", "go get github.com/gobuffalo/packr"
+    system "./goget_packr.sh"
```

Comment:
could this be `go mod download` instead?

Reply:
Maybe? I'm just copying what the go 1.16 changes said to do.

https://blog.golang.org/go116-module-changes

I'm not sure if go mod download fixes things with 1.16. let me know what you think we should do

Reply:
I'd rather try and avoid tricking the audit. Possible solutions in my book:
- Figure out how to make Go create this in a mod file
- Manually add a modfile here

Reply:
I had a look at this. Looks like the formula is also missing the newer `packr2` binary. This is what I did to get it working: #71748.